source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -390,15 +391,15 @@ def filter_func(self, journey):
         return max_duration > journey.duration
 
 
-def get_min_connections(journeys):
+def get_best_pt_journey_connections(journeys, request):
     """
-    Returns min connection count among journeys
+    Returns the nb of connection of the best pt_journey
     Returns None if journeys empty
     """
     if not journeys:
         return None
-
-    return portable_min((get_nb_connections(j) for j in journeys if not to_be_deleted(j)), default=0)
+    best = get_ASAP_journey((j for j in journeys if 'non_pt' not in j.tags), request)
+    return get_nb_connections(best) if best else 0
```

Comment:
```suggestion
    return get_nb_connections(best) if best else None
```
otherwise the following would not work isn't it ? `if best_pt_journey_connections is not None:`